Car.java

```diff
@@ -0,0 +1,94 @@
+public class Car {
+
+    private String color;
+    private String mark;
+    private int consumption;
+    private double volume;
+    private double mileage;
+
+
+    public Car(String color, String mark, int cons, double vol, double mil) {
+        this.color = color;
+        this.mark = mark;
+        this.consumption = cons;
+        this.volume = vol;
+        this.mileage = mil;
+    }
+
+
+    public void drive(int km) {
+
+        double volumeForTheTrip = consumption * km;
+        double fuelLeft = volume - volumeForTheTrip;
+
+
+        if (fuelLeft > 0) {
+            System.out.println("Проехала: " + km + " км\n");
+            System.out.println("Пробег: " + (km + mileage) + "км\n");

```

Comment:
You do 'mileage + km' action twice. It can easy be done once and displayed after that.